x.py

```diff
@@ -266,6 +266,7 @@ def test_cpp(dir: str, rest: List[str]) -> None:
 
 def test_go(dir: str, rest: List[str]) -> None:
     go = find_command('go', msg='go is required for testing')
+    find_command('redis-cli', msg='redis-cli is required for testing')
```

Comment:
```suggestion
def test_go(dir: str, cli_path: str, rest: List[str]) -> None:
    go = find_command('go', msg='go is required for testing')
    find_command(cli_path, msg='redis-cli is required for testing')
```

To respect `cliPath` according to @tisonkun , you can follow these proposed changes:

And add this line:
```
parser_test_go.add_argument('--cli-path', default='redis-cli', help="path of redis-cli to test kvrocks")
```

Append `cliPath` to `args`:
```
args = [
        'test', '-bench=.', './...',
        f'-binPath={binpath}',
        f'-cliPath={cli_path}',
        f'-workspace={worksapce}',
        *rest
    ]
```